src/server/graphql/models/Action/actionMutation.js

```diff
@@ -68,5 +69,55 @@ export default {
       };
       await r.table('Action').insert(action);
     }
+  },
+  deleteAction: {
+    type: GraphQLBoolean,
+    description: 'Delete an action',
+    args: {
+      actionId: {
+        type: new GraphQLNonNull(GraphQLID),
+        description: 'The actionId (teamId::shortid) to delete'
+      }
+    },
+    async resolve(source, {actionId}, {authToken}) {
+      // format of id is teamId::taskIdPart
+      const [teamId] = actionId.split('::');
+      requireSUOrTeamMember(authToken, teamId);
+      await r.table('Action').get(actionId).delete();
+    }
+  },
+  makeProject: {
```

Comment:
Wow. You decided to make the project->action, action->project transformers. That was extra credit! Thank you!